components/raftstore/src/store/snap.rs

```diff
@@ -1612,6 +1632,34 @@ impl SnapManager {
         Ok(Box::new(f))
     }
 
+    // Tablet snapshot is the snapshot sent from raftstore-v2.
+    // We enable v1 to receive it to enable tiflash node to receive and apply
+    // snapshot from raftstore-v2.
+    // To make it easy, we maintain an empty `store::snapshot` with tablet snapshot
```

Comment:
So the only useful information in this empty snapshot is tablet snapshot path, right? Then the most of the logic in new_for_tablet_snapshot may not be necessary, like 
" ```
s.init_for_building()?;
   let mut meta = gen_snapshot_meta(&s.cf_files[..], for_balance)?;
```"